packages/design-system/src/styles/components/_Tooltip.scss

```diff
@@ -138,9 +126,9 @@ $tooltip-close-min-width: $spacer-4 !default;
 .ds-c-tooltip {
   background: $tooltip__background-color;
   border-color: $tooltip__border-color;
-  border-radius: $tooltip-border-radius;
+  border-radius: $radius-large;
   border-style: solid;
-  border-width: $tooltip-border-width;
```

Comment:
This one seems like it should be a theme variable just because it is repeated so often and seems a shame to turn into a magic number.